qiskit_experiments/data_processing/processor_library.py

```diff
@@ -98,3 +116,66 @@ def get_processor(
         return processor
 
     raise DataProcessorError(f"Unsupported measurement level {meas_level}.")
+
+
+def is_restless(experiment_data: ExperimentData, analysis_options: Options, index: -1) -> bool:
```

Comment:
It would be good to test this function more thoroughly. Currently, the new end to end tests test that it does get turned on for certain parameters and some of the other tests in the test suite must be testing that it is not turned on for some sets of parameters but it would be good to test the various cases that turn it off.